pkg/backend/filestate/backend.go

```diff
@@ -104,10 +106,14 @@ func New(d diag.Sink, url string) (Backend, error) {
 		return nil, errors.Wrapf(err, "unable to open bucket %s", url)
 	}
 
+	lockID := uuid.NewV4().String()
+	lockID = strings.Split(lockID, "-")[0]
```

Comment:
any reason we're just grabbing the first portoin, as opposed to using the entire uuid?

Reply:
It was shortened to be more in line with the shorter uuids generated for Pulumi resources. I have no preference either way.